modules/tinymce/src/themes/silver/main/ts/ui/window/SilverDialogEvents.ts

```diff
@@ -95,7 +95,7 @@ const initDialog = <T extends Dialog.DialogData>(getInstanceApi: () => Dialog.Di
     }),
 
     fireApiEvent<FormActionEvent>(formActionEvent, (api, spec, event, component) => {
-      const focusIn = component.getSystem().isConnected() ? () => Keying.focusIn(component) : Fun.noop;
+      const focusIn = () => component.getSystem().isConnected() ? Keying.focusIn(component) : Fun.noop();
```

Comment:
I feel this change is radical. There should be tests to cover this logic change

Reply:
that was the original method, there was a suggestion to change to the current one, which breaks the change

Reply:
the focus should be focus on the dialog when event is fired, there is already test that covers the dialog focus.

Reply:
Is there a test that fails using the breaking method? In case someone attempts to change that in the future.

Reply:
didn't see any test breaking, let me run again

Reply:
It would be great to have a test that fails with the breaking change. If it's not convenient to add it now, I recommend creating a new task ticket to add such a test in the future for 6.7.

Reply:
looks fine on my local machine, I attached a jira ticket for the test creation